### DISCUSSION TITLE
Workarounds for current Gradio issues

### DISCUSSION THREAD

**Discussion by AlUlkesh** [AlUlkesh/stable-diffusion-webui-images-browser#107] [Announcements]
Different people are having different issues since webui switched to the new Gradio. This includes webui itself and extensions.

What you can try:

- First, make sure you have the latest commits of both webui and this extension
- You can install this PR, while it's waiting to be merged: https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/9060
- You can try a different browser, Chrome/Firefox

**Comment by SwannSchilling** [comment]
Worked for me!! Thanks a lot!!

**Comment by AlUlkesh** [comment]
For anyone who wants to try the PR mentioned in the second point above:

You can create a local branch like this (from the webui directory):

```
git fetch origin pull/9060/head:pr9060
git checkout pr9060
```

And if you want to switch back to the master branch later, do:

`git checkout master`
